src/components/NftTileDetails/NftTileDetails.tsx

```diff
@@ -173,9 +173,17 @@ export const NftTileDetails: React.FC<NftTileDetailsProps> = ({
           !buyNow && <DetailsContent caption="Status" content="Not for sale" icon={<SvgActionNotForSale />} secondary />
         )
       case 'minBid':
-        return <DetailsContent caption="Min bid" content={minBid || ''} icon={<SvgActionJoyToken />} />
+        return (
+          minBid && (
```

Comment:
This change could be bug-prone. If I set the auction to `minBid` and `minBid` to `0` I'll see: 
![image](https://user-images.githubusercontent.com/51168865/150328981-1f5e343d-8f4c-44e4-92dd-c59fb8f4f3fd.png)